app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -117,37 +116,6 @@ def installable_errata
                                                      :resource_class => Erratum))
     end
 
-    api :PUT, "/hosts/bulk/install_content", N_("Install content on one or more hosts using katello-agent. %s") % katello_agent_deprecation_text, deprecated: true
```

Comment:
Need to remove from => https://github.com/Katello/katello/blob/master/config/routes/overrides.rb#L72

Reply:
Basically you need to do this for every action you removed.